templates/example_visualization.ipynb

```diff
@@ -0,0 +1,224 @@
+{
```

Comment:
<p>Typo in the comment</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/angelolab/ark-analysis/pull/171/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='8'/>